src/WormholeReceive.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+abstract contract WormholeReceive {
```

Comment:
Nice I like pulling out these utility messages. Should we also leverage this in the token bridging contracts, then?

Reply:
Good suggestion added https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/b7f3ac5d29509fd00ae44607171b50d5f8df8b1f